tests/Aspire.Hosting.Tests/ManifestGenerationTests.cs

```diff
@@ -136,6 +136,50 @@ public void ExcludeLaunchProfileOmitsBindings()
             "Service has no bindings because they weren't populated from the launch profile.");
     }
 
+    [Fact]
+    public void ProjectResourceEmitsForwardedHeadersEnvironmentVariableByDefault()
+    {
+        var appBuilder = DistributedApplication.CreateBuilder(new DistributedApplicationOptions
+        { Args = GetManifestArgs(), DisableDashboard = true, AssemblyName = typeof(ManifestGenerationTests).Assembly.FullName });
+
+        appBuilder.AddProject<Projects.ServiceA>("servicea");
+
+        appBuilder.Services.AddKeyedSingleton<IDistributedApplicationPublisher, JsonDocumentManifestPublisher>("manifest");
+
+        var program = appBuilder.Build();
+        var publisher = program.Services.GetManifestPublisher();
+
+        program.Run();
+
+        var resources = publisher.ManifestDocument.RootElement.GetProperty("resources");
+
+        Assert.Equal("true",
+            resources.GetProperty("servicea").GetProperty("env").GetProperty("ASPNETCORE_FORWARDEDHEADERS_ENABLED").GetString());
```

Comment:
Given we're just verifying a single property in the manifest in these tests I did it via the JSON APIs rather than comparing the whole manifest. The update in `ProjectResourceTests.cs` verifies the whole manifest.